update-urls/main.go

```diff
@@ -61,9 +61,6 @@ var (
 		"AppsService.FindRepositoryInstallationByID": true,
 		"AuthorizationsService.CreateImpersonation":  true,
 		"AuthorizationsService.DeleteImpersonation":  true,
-		"GitService.GetRef":                          true,
```

Comment:
Uh, wow... that you for updating my maintenance tool for me as well!
I really appreciate that!  :heart: